tests/unit/test_options_config.py

```diff
@@ -218,3 +218,35 @@ def test_parse_config_ignores_unknowns(tmp_path, opt_manager, caplog):
 def test_load_config_missing_file_raises_exception(capsys):
     with pytest.raises(exceptions.ExecutionError):
         config.load_config("foo.cfg", [])
+
+
+def test_invalid_ignore_codes_raise_error(tmpdir, opt_manager):
+    tmpdir.join("setup.cfg").write("[flake8]\nignore = E203, //comment")
+    with tmpdir.as_cwd():
+        cfg, _ = config.load_config("setup.cfg", [], isolated=False)
+
+    with pytest.raises(ValueError) as excinfo:
+        config.parse_config(opt_manager, cfg, tmpdir)
+
+    expected = (
+        "Error code //comment supplied to 'ignore' option "
+        "does not match ^[A-Z]{1,3}[0-9]{0,3}$"
+    )
+    msg = excinfo.value.args[0]
+    assert msg == (expected)
```

Comment:
why didn't you use the code I wrote for you in the comment?  this assertion is weaker than the one I wrote for you as it allows an exception with more arguments to pass silently

also these parens do nothing

Reply:
Hi, sorry about being quiet for a while but I caught a nasty covid case. Fixed this assertion now!